src/integration/integration_api_scheduler_test.go

```diff
@@ -94,6 +122,34 @@ var _ = Describe("Integration_Api_Scheduler", func() {
 				resp.Body.Close()
 
 			})
+
+			Context("Scheduler is down", func() {
+				BeforeEach(func() {
+					stopScheduler(schedulerProcess)
+
+					policyStr = readPolicyFromFile("fakePolicyWithoutSchedule.json")
+					resp, err := attachPolicy(appId, policyStr)
+					Expect(err).NotTo(HaveOccurred())
+					Expect(resp.StatusCode).To(Equal(http.StatusCreated))
+					resp.Body.Close()
+
+					schedulerProcess = startScheduler()
```

Comment:
AfterEach

I would restructure the tests slightly.
Pull the "Scheduler is down" up, and really we only need 1 test since with or without schedule should affect the outcome but I would do it with a schedule since that would be something the scheduler participates in.